### PR TITLE
[Connection Management] Vault: fix sync chip too long

### LINKED ISSUE #7610: [Bug] Error chip overflows connection table
Image

### PR DESCRIPTION
**Pull request by philipperolet**
Description
---
Fixes https://github.com/dust-tt/dust/issues/7610

Not trivial to make tooltip work after trying 15mns. Going for just a shorter message.

Risk & deploy
---
na

### CODE REVIEW

File: front/components/vaults/VaultResourcesList.tsx [hunk 1]
```diff
@@ -167,9 +167,7 @@ const getTableColumns = ({
             !info.row.original.dataSourceView.dataSource
               .fetchConnectorError && <Chip color="amber">Never</Chip>}
           {info.row.original.dataSourceView.dataSource.fetchConnectorError && (
-            <Chip color="warning">
-              Error loading the connector. Try again in a few minutes.
-            </Chip>
+            <Chip color="warning">Retry in a few minutes</Chip>
```
Comment (flvndvd) [inline front/components/vaults/VaultResourcesList.tsx:170]: The proper fix should probably be in `DataTable` to have flexible columns in their, right 🙃.

Comment (philipperolet) [reply front/components/vaults/VaultResourcesList.tsx:170]: Respectfully disagree, we don't want the sync chip to be too long so the proper fix is to keep it short.

Comment (philipperolet) [reply front/components/vaults/VaultResourcesList.tsx:170]: Making a functional tooltip would be the full proper fix IMO (but tougher job)

Comment (flvndvd) [reply front/components/vaults/VaultResourcesList.tsx:170]: It's not a tooltip here but a Chip, right?

Comment (philipperolet) [reply front/components/vaults/VaultResourcesList.tsx:170]: It's a chip, and the "Try again in a few minutes" should be in a tooltip when you hover the chip

Comment (flvndvd) [reply front/components/vaults/VaultResourcesList.tsx:170]: 👍 Let's merge this until we refactor the table.

**Review by flvndvd — APPROVED** [review]

**Pull request merged**